english/lego/lego.md

```diff
@@ -22,10 +22,12 @@ or
 
 [Bake a cake?](../dance/bake-a-cake/bake-a-cake.md)
 
-or 
+or
 
 [Look for a way out?](../find-exit/leave.md)
 
 or
 
 [Ponder](../ponder/ponder-about-life.md)
+
+or [Kick the man](../kicktheman/kick-the-man.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.